### PR TITLE
给message组件添加stack全局配置

### LINKED ISSUE #3543: 多条message的显示问题
调用了多条message时现在会覆盖 是否可以考虑纵向排列避免覆盖

### PR DESCRIPTION
**Pull request by yeliex**
First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!
- [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
- [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
- [x] Add some descriptions and refer relative issues for you PR.

Close: #3543

往message.config中添加了一个stack配置,允许设置message组件重叠显示还是排列 
已添加demo

### CODE REVIEW

**Comment by benjycui** [comment]
CI failed, `strictNullChecks` is enable now.

**Comment by yeliex** [comment]
@benjycui 修复了

File: components/message/demo/unstack.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+order: 4
+title:
+  zh-CN: 不堆叠
+  en-US: unstack
+---
+
+## zh-CN
+
+使每条message从上到下排列
```
Comment (afc163) [inline components/message/demo/unstack.md]: - 中英文之间应该有空格。
- 应该有句号。

**Comment by yeliex** [comment]
本地`npm run test`通过 ci 报错不知道为什么

File: components/message/index.tsx [hunk 2]
```diff
@@ -94,6 +96,9 @@ export default {
     if (options.prefixCls !== undefined) {
       prefixCls = options.prefixCls;
     }
+    if ('stack' in options) {
+      defaultStack = options.stack || defaultStack;
```
Comment (benjycui) [inline components/message/index.tsx]: ???

这样就永远都是 `true` 了。

**Comment by benjycui** [comment]
猜测是 Node 版本问题，已经更新 travis.yml，麻烦 rebase 一下看看。

File: components/message/index.tsx [hunk 3]
```diff
@@ -7,10 +7,11 @@ let defaultTop;
 let messageInstance;
 let key = 1;
 let prefixCls = 'ant-message';
+let defaultStack = true;
 
 function getMessageInstance() {
   messageInstance = messageInstance || Notification.newInstance({
-    prefixCls,
+    prefixCls: `${!defaultStack ? `${prefixCls}-unstack ` : ''}${prefixCls}`,
```
Comment (benjycui) [inline components/message/index.tsx]: `prefixCls` 被当成 `className` 来用了 😂😂

**Comment by benjycui** [comment]
FYI https://github.com/ant-design/ant-design/commit/75423ed4b1819f88e5aeb1b2154f572aea2da2df

样式还是有点问题，message 两边看不到的地方覆盖了后面的元素。

可能需要修改 rc-notification，我先把 demo 隐藏起来，有好的想法在改。

**Pull request merged**